index.html

```diff
@@ -265,20 +263,36 @@ <h4 class="card-title">
       </div>
       <!-- /.row -->
 
+      <hr>
+
+      <!-- Call to Action Section -->
+      <div class="row mb-4">
+        <div class="col-md-8">
+          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Molestias, expedita, saepe, vero rerum deleniti
+            beatae veniam harum neque nemo praesentium cum alias asperiores commodi.</p>
+        </div>
+        <div class="col-md-4">
+          <a class="btn btn-lg btn-secondary btn-block" href="#">Call to Action</a>
+        </div>
+      </div>
+
+      </div>
+      <!-- /.container -->
+
       <!-- Comments-->
       <div class="col-lg-4 col-sm-6">
-        <h5 class="card-header">Leave a Comment:</h5>
         <div class="card-body">
           <form>
             <div class="form-group">
-              <textarea class="form-control" rows="3"></textarea>
+              <label class="sr-only"  for="Textarea">Add comment</label>
```

Comment:
I like this comment box. I was wondering what the team thinks about having it more prominent, off to the left at the top for example.

Reply:
I like the Idea. I am also thinking of removing the "class="sr-only" looks like it is preventing the Add comment from displaying."